test/browser/run-test.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+set -eux
+
+# tests need cockpit's bots/ libraries and test infrastructure
+cd $SOURCE
+git init
+make bots test/common
+
+# only install a subset to save time/space
+rm -f package-lock.json  # otherwise the command below installs *everything*, argh
+npm install chrome-remote-interface sizzle
+
+. /etc/os-release
+export TEST_OS="${ID}-${VERSION_ID/./-}"
+export TEST_AUDIT_NO_SELINUX=1
+
+EXCLUDES=""
+
+RC=0
+test/common/run-tests --nondestructive --machine 127.0.0.1:22 --browser 127.0.0.1:9090 $EXCLUDES || RC=$?
```

Comment:
This did not actually [run any tests](http://artifacts.dev.testing-farm.io/cb28e9e1-9e78-4aa3-8ad3-804a59695eea/), presumably because c-certificates does not currently have nondestructive tests?

Reply:
Good point - will be my next step here. This and releases.